components/ActivityIndicator/examples/supplemental/ActivityIndicatorExampleSupplemental.m

```diff
@@ -83,7 +83,7 @@ - (void)setupExampleViews {
 
   CGRect sliderFrame = CGRectMake(0, 0, 160, 27);
   self.slider = [[UISlider alloc] initWithFrame:sliderFrame];
-  self.slider.value = kActivityInitialProgress;
+  self.slider.value = (float)kActivityInitialProgress;
```

Comment:
could we just make `kActivityInitialProgress` a `float` in its declaration in the header file?

Reply:
I've decided against this after thinking it through some more. If we really think the refactoring is important, we can do it in a separate PR.

Reply:
fair